includes/abilities-api.php

```diff
@@ -35,8 +35,8 @@
  *   description?: string,
  *   input_schema?: array<string,mixed>,
  *   output_schema?: array<string,mixed>,
- *   execute_callback?: callable( array<string,mixed> $input): (mixed|\WP_Error),
- *   permission_callback?: callable( array<string,mixed> $input ): (bool|\WP_Error),
+ *   execute_callback?: callable( mixed $input ): (mixed|\WP_Error),
+ *   permission_callback?: callable( mixed $input ): (bool|\WP_Error),
```

Comment:
We need to change these to allow the callbacks to not expect any `$input` (relevant when `input_schema` is empty).

Reply:
Good point. It should be optional to define the `$input` based on the conversations. I'll update these type hints accordingly 👍🏻

Reply:
Addressed in https://github.com/WordPress/abilities-api/pull/61/commits/75d89cfb985d4908f5a36665c1d234bd30f886c3.